lib/stdlib/test/gen_event_SUITE.erl

```diff
@@ -1345,20 +1345,23 @@ format_log_1(_Config) ->
     Depth = 10,
     ok = application:set_env(kernel, error_logger_format_depth, Depth),
     Limited = [1,2,3,4,5,6,7,8,9,'...'],
+    LastMsg = ["Last msg" | Limited],
+    State   = ["State" | Limited],
+    Reason  = ["Reason" | Limited],
```

Comment:
The test case fails as it is now and I don't think it tests what it should.

I haven't tested this change, but it should work.

```suggestion
    Limited = [1,2,3,4,5,6,7,8,'...'],
    LastMsg = ["Last msg" | Term],
    State   = ["State" | Term],
    Reason  = ["Reason" | Term],
```